src/main/java/wile/engineersdecor/blocks/EdCatwalkBlock.java

```diff
@@ -62,7 +62,7 @@ public BlockState getStateForPlacement(BlockPlaceContext context)
 
   public static boolean place_consume(BlockState state, Level world, BlockPos pos, Player player, InteractionHand hand, int shrink)
   {
-    if(!world.setBlock(pos, state, 1|2)) return false;
+    if(!world.setBlock(pos, state, Block.UPDATE_ALL)) return false;
```

Comment:
Minecraft added these magic constants to the `Block` class since 1.17, now there's no need to memorize them or look up in Forge constants helper.